guides/v2.3/graphql/reference/customer.md

```diff
@@ -74,6 +79,18 @@ Attribute |  Data Type | Description
 `region` | String | The state or province name
 `region_id` | Int | Uniquely identifies the region
 
+### isEmailAvailable query attribute
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`email` | String! | The email address to check
+
+### IsEmailAvailableOutput attribute
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`is_email_available` | Boolean | A value of `true` indicates the email address can be used to create an account
```

Comment:
Depending on my above question, this might need some rewriting. A value of `true` indicates the email address is available to use to create an account....or something like that.